src/styles/index.scss

```diff
@@ -1,3 +1,166 @@
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+  background-color: #f4f4f4;
+}
+
+.product {
+  display: flex;
+  justify-content: center;
+
+  &-image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &-card {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    background-color: var(--white);
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+  }
+
+  &-image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &-title {
+    color: var(--main-accent);
+    font-family: "Roboto", sans-serif;
```

Comment:
Consider setting the font family for the entire page to get elements inherited from it and avoid repeating code